tests/components/notify/test_file.py

```diff
@@ -29,7 +29,7 @@ def test_bad_config(self):
             'notify': {
```

Comment:
This seems to pass for me with the change to `notify/_init_.py`:

```python
-        self.assertFalse(notify.setup(self.hass, {
+        self.assertFalse(_setup_component(self.hass, 'notify', {
```

and import on top: `from homeassistant.bootstrap import _setup_component`